pages/common/telnet.md

```diff
@@ -1,19 +1,19 @@
 # telnet
 
-> Telnet is used to connect to a specified port of a host.
+> Connect to a specified port of a host using the telnet protocol.
 
 - Telnet to a certain port:
```

Comment:
While we are on this page, maybe we should start with a simple 'telnet {{host}}' example and say that telnet connects to port 23 by default.